bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wprimetotb/Wprimetotb_LR_customizecards.dat

```diff
@@ -0,0 +1,5 @@
+set MADSPIN=ON
```

Comment:
That should be correctly handled by our scripts. Did you had problems when not having this line?

Reply:
I don't think so, but we already produced GP with this line in

Reply:
do you mind removing from the pr. just not to cause confusion. olivier added this because he doesn't know about our scripts.

Reply:
Done